src/seedu/addressbook/data/person/Person.java

```diff
@@ -80,8 +78,6 @@ public int hashCode() {
     }
 
     @Override
-    public String toString() {
-        return getAsTextShowAll();
-    }
+    public String toString() { return getPrintableString(getName(), getPhone(), getEmail(), getAddress()); }
```

Comment:
Single line methods are good in reducing code length. However, it is a disaster when it comes to readability.
It take a few moments for people to understand the statement in your method and it is prompt to hidden statements such as this
> public void do something() { a; b; c; }

If the statement is too lengthy/messy people may assume that there is only 1 statement (a) when it actually has 3 (a,b,c)